src/services/mongoose/schema.types.d.ts

```diff
@@ -163,3 +163,18 @@ type PaymentFlowStateRecordIndex = XOR<
   senderWalletId: string
   inputAmount: number
 }
+
+type WalletOnChainPendingReceiveRecord = {
+  walletId: string
+  address: string
+  transactionHash: string
```

Comment:
We should probably add `vout` as well

Reply:
Probably we will need to fix the vout issue first in current implementation but... I added it https://github.com/GaloyMoney/galoy/pull/2622/commits/903ca44ded470fca59c481cbf8fcebccb121948a